PULL_REQUEST_TEMPLATE

```diff
@@ -1,24 +1,32 @@
 Jira: [JIRA-TOKEN](https://ticketevolution.atlassian.net/browse/JIRA-TOKEN)
 
+> [!IMPORTANT]
+> Toggle to SQUASH AND MERGE when merging PRs to `staging` branch
+
 ## Why / What
+Why: These changes are for...
+
+What: This PR adds/removes/fixes... 
 
-This PR adds/removes/fixes&hellip;
+If this PR requires other developers to make changes to their local environment after this PR has been merged, please
+describe here. Examples: DB migration, Docker container restart/rebuild, .env changes etc.
 
 ## Testing
 
-I tested this code by&hellip;
+I tested this code by (unit/integration/manual) testing...
+
+## Environment Variables
+
+If there are any changes to environment variables, list here and link any relevant `tevo-kubernetes` PRs. Update the 
+project's .env file in 1Password after PR has been merged if necessary.
 
-## Reviewer Checklist (To be edited and marked by approver)
+## Database
 
-After approving the PR, the reviewer should check the boxes below. See the [Reviewer Checklist](https://ticketevolution.atlassian.net/wiki/spaces/EN/pages/3015016552/Reviewer+Checklist) company standard for more info on these bullet points.
+If any DB changes, describe here. Add DBAs to the PR reviewers to approve DB migrations
 
-- [ ] Do you understand the reasoning behind the need for the changes?
-- [ ] Given only new or existing tests*, do you feel confident that all changes work as intended and solve the problem at hand?
-- [ ] Are all success and failure paths accounted for in the tests? If not, are the omissions explained/justified?
-- [ ] If coverage ignores are present, is justification provided inline (or linked to) or is a link to a ticket to address in the future provided?
-- [ ] Are you willing to accept the same responsibility for this code as the author?
+## UI
 
-*\*If existing tests, requester should point out what applies*
+If this PR has visual changes, add screenshots or videos here
```

Comment:
I like this, there are often screenshots needed 🎉

Reply:
👀 Matt you misspelled "never"